src/nih_wayfinding/app/scripts/mapping/map-control-service.js

```diff
@@ -1,9 +1,10 @@
-(function () {
+(function (_) {
```

Comment:
Hmm, explicitly including this dependency in this way is an interesting idea. Might explore this in the future, but I would remove it for now so that it is consistent with the rest of the project.